dags/pytorch_xla/configs/pytorchxla_torchbench_config.py

```diff
@@ -242,6 +273,17 @@ def get_nvidia_driver_install_cmd(driver_version: str) -> str:
     )
     return nvidia_driver_install
 
+  install_torch_xla2_dependency = (
+      (
+          # TODO(piz): torch_xla2 only support nightly test at this time.
+          "pip install torch_xla[pallas] -f https://storage.googleapis.com/jax-releases/jax_nightly_releases.html -f https://storage.googleapis.com/jax-releases/jaxlib_nightly_releases.html",
+          "cd /tmp/xla/experimental/torch_xla2/",
```

Comment:
Do you need to clone this? You should be able actually install directly from git: https://pip.pypa.io/en/stable/topics/vcs-support/#url-fragments

That's one advantage of a pure python project. That wouldn't work for the main `torch_xla` package.

Reply:
Oh, good to know we can install directly.

We still need scripts under xla/benchmarks/..., so we need to clone the repo anyway.